src/encoded/static/components/genome_browser.js

```diff
@@ -911,15 +913,22 @@ class GenomeBrowser extends React.Component {
                 labelLength += file.assay_term_name ? file.assay_term_name.length + 2 : 0;
                 labelLength += file.biosample_ontology && file.biosample_ontology.term_name ? file.biosample_ontology.term_name.length + 2 : 0;
                 labelLength += file.annotation_subtype ? file.annotation_subtype.length : 0;
+
+                extLabelLength += file.output_type ? file.output_type.length : 0;
+                extLabelLength += file.simple_biosample_summary ? file.simple_biosample_summary.length : 0;
+                extLabelLength += file.biological_replicates ? file.biological_replicates.length : 0;
+                extLabelLength += file.biochemical_inputs ? file.biochemical_inputs.length : 0;
```

Comment:
`biological_replicates` and `biochemical_inputs` are both arrays. `biological_replicates` array length and combined character length are probably not too far off, but `biochemical_inputs` array length and combined character length can be pretty far off. I’m not sure if, since this is a rough estimate of character length (and `defaultExtHeight` is an _extremely_ rough minimal guess), you figured this was just part of that roughness.

I’ll assume you meant to do this, and you can change it if you didn’t mean to.